app/views/account/companies/show.html.slim

```diff
@@ -67,3 +67,18 @@
           .card-content
             .card-body
               = render 'account/companies/html/social_links', company: @company
+        - unless @tweets.blank?
+          .card
+            .card-header
+              .row
+                .col-md-6
+                  h4.card-title= "#{@company.name} tweets"
+                  h5.card-title
+                    =link_to @company.twitter, target: '_blank' do
+                      | @#{@tweets[0].user.screen_name}

```

Comment:
create in company method "twitter_user_name" and use here and in your processor

I DON"T LIKE code like `[0]`

Reply:
OR

you can make screen_name as a part of TwiiterProcessor class
@processor = ....

and then in view
@processor.username
or
@processor.followers_count
or
@processor.tweets.each